elasticsearch/Dockerfile

```diff
@@ -1,7 +1,8 @@
 ARG ELK_VERSION
+ARG ELK_FLAVOUR
 
-# https://github.com/elastic/elasticsearch-docker
-FROM docker.elastic.co/elasticsearch/elasticsearch-oss:${ELK_VERSION}
+# https://github.com/elastic/elasticsearch-docker / https://www.docker.elastic.co/
```

Comment:
https://www.docker.elastic.co/ is only a list of images, not so relevant in the context IMO.

Reply:
changed with f88c8c3b292c36d19e34f8146ef356cad3ac0738